Resources/Prototypes/Nyanotrasen/Catalog/Fills/Vending/Inventories/maildrobe.yml

```diff
@@ -0,0 +1,8 @@
+- type: vendingMachineInventory
+  id: RepDrobeInventory
+  startingInventory:
+    ClothingUniformJumpsuitMailman: 2
+    ClothingUniformJumpskirtMailman: 2
+    ClothingHeadHatsMailman: 2
+    ClothingBeltMailbag: 2
+    ClothingEarsHeadsetCargo: 2
```

Comment:
All of the repdrobe issues was because I was messing around and trying to figure out how vending machines worked in yaml. It seems I got sloppy with my cleanup.